jlibra-core/pom.xml

```diff
@@ -18,6 +18,11 @@
       <artifactId>jlibra-grpc</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>futures-extra</artifactId>
+      <version>4.2.0</version>
```

Comment:
I checked from the repo that the latest version is 4.2.2, maybe we could go to that directly?

Reply:
done